packages/@aws-cdk/aws-appmesh/test/integ.mesh.ts

```diff
@@ -92,6 +99,13 @@ const node3 = mesh.addVirtualNode('node3', {
       unhealthyThreshold: 2,
     },
   })],
+  backendDefaults: {
+    tlsClientPolicy: {
+      validation: appmesh.TLSValidationContext.fileTrust({
```

Comment:
We're testing `fileTrust` twice here. Should we use a different static factory method for the second time?